compositions/Header/Header.css.ts

```diff
@@ -1,15 +1,15 @@
-import { style, globalStyle } from '@vanilla-extract/css'
-import { atoms, media, color } from '@zoralabs/zord'
+import { style } from '@vanilla-extract/css'
+import { atoms, media, color, space } from '@zoralabs/zord'
 import { HEADER_HEIGHT, HEADER_HEIGHT_MOBILE, HEADER_Z } from 'styles/style-constants'
 
 export const headerWrapper = style([
   {
     height: HEADER_HEIGHT_MOBILE,
     zIndex: HEADER_Z,
-    alignContent: 'center',
     alignItems: 'center',
     gridTemplateColumns: 'repeat(2, 1fr)',
     gridTemplateRows: 'auto',
+    justifyContent: 'start',
```

Comment:
alignItems and justifyContent can be atomized

Reply:
actually doesn't need to be there at all - but we don't actually have 'start' in zord - just 'flex-start'
And i believe start applies to usage with css grid.
probably worth me looking further into why this surfaced.
and makes me wonder - for any css properties we have in zord - should we follow the standard spec and include all options?

<img width="797" alt="Screen Shot 2022-08-19 at 4 50 06 PM" src="https://user-images.githubusercontent.com/7268786/185720503-49d1592e-faca-4f76-b117-97f6169758e2.png">